src/client/js/otp/modules/planner/PlannerModule.js

```diff
@@ -107,12 +105,6 @@ otp.modules.planner.PlannerModule =
         
         this.defaultQueryParams = _.clone(otp.modules.planner.defaultQueryParams);
 
-        if (otp.config.metric) {
```

Comment:
Per https://github.com/CUTR-at-USF/usf-mobullity/issues/98#issuecomment-149992750, I think this block should stay - it still exists upstream.